scripts/algorithms/l2-svm.dml

```diff
@@ -35,7 +35,7 @@
 #
 
 cmdLine_fmt = ifdef($fmt, "text")
-cmdLine_icpt = ifdef($icpt, 0)
+cmdLine_icpt = ifdef($icpt, 1)
```

Comment:
please don't change the default intercept as it might surprise people w/o explicit specification.